conf/modules.config

```diff
@@ -474,7 +474,15 @@ process {
             path: { "${params.outdir}/${meta.id}/annotation-summary" },
             mode: params.publish_dir_mode,
             pattern: "*summary.gff.gz",
-            saveAs: { filename -> filename.equals('versions.yml') ? null : filename },
+        ]
+    }
+
+    withName: "GT_GFF3VALIDATOR" {
+        publishDir = [
```

Comment:
maybe we should also publish `success.log` to make sure gt_validator has done its job?

Reply:
I would skip that.. it's just an empty file. No error file == success :)